crates/tools/src/main.rs

```diff
@@ -122,7 +122,7 @@ fn install_code_extension() -> Result<()> {
         run(r"npm install", "./editors/code")?;
     }
     run(
-        r"node ./node_modules/vsce/out/vsce package",
+        r"npm run package",
```

Comment:
Does this actually work? How this calles into `vsce` thigs?

That is, does `cargo install-code` still installes the extension itself?